requirements.txt

```diff
@@ -1,3 +1,4 @@
-nibabel>=3.2.1
+nibabel >= 3.2.1 ; python_version > "3.0"
```

Comment:
Is strictly greater the typical test for the python version?  Not that someone is likely to be running exactly python 3.0, but...

Reply:
Oops... thanks for catching that. I'll fix it.